main.qml

```diff
@@ -453,7 +478,7 @@ ApplicationWindow {
         currentWallet.pauseRefresh();
 
         appWindow.showProcessingSplash(qsTr("Waiting for daemon to start..."))
-        daemonManager.start(flags, persistentSettings.testnet, persistentSettings.blockchainDataDir, persistentSettings.bootstrapNodeAddress);
+        daemonManager.start(flags, persistentSettings.testnet, persistentSettings.blockchainDataDir);
```

Comment:
Is this correct?

Reply:
Sorry, not correct. Must've got lost in the (many) rebase(s)...

Reply:
Fixed